docs/pages/includes/metrics.mdx

```diff
@@ -1,59 +1,67 @@
+| `firestore_events_backend_write_seconds`                 | histogram | GCP Cloud Firestore | Latency for Cloud Firestore events write operations.                                               |
+| `gcs_event_storage_downloads_seconds`                    | histogram | GCP GCS             | Latency for GCS download operations.                                                               |
+| `gcs_event_storage_downloads`                            | counter   | GCP GCS             | Number of downloads from the GCS backend.                                                          |
+| `gcs_event_storage_uploads_seconds`                      | histogram | GCP GCS             | Latency for GCS upload operations.                                                                 |
+| `gcs_event_storage_uploads`                              | counter   | GCP GCS             | Number of uploads to the GCS backend.                                                              |
+| `grpc_server_started_total`                              | counter   | Teleport Auth       | Total number of RPCs started on the server.                                                        |
+| `grpc_server_handled_total`                              | counter   | Teleport Auth       | Total number of RPCs completed on the server, regardless of success or failure.                    |
+| `grpc_server_msg_received_total`                         | counter   | Teleport Auth       | Total number of RPC stream messages received on the server.                                        |
+| `grpc_server_msg_sent_total`                             | counter   | Teleport Auth       | Total number of gRPC stream messages sent by the server.                                           |
+| `heartbeat_connections_received_total`                   | counter   | Teleport Auth       | Number of times the Auth Service received a heartbeat connection.                                  |
+| `s3_requests_total`                                      | counter   | Amazon S3           | Total number of requests to the S3 API.                                                            |
+| `s3_requests`                                            | counter   | Amazon S3           | Total number of requests to the S3 API grouped by result.                                          |
+| `s3_requests_seconds`                                    | histogram | Amazon S3           | Request latency for the S3 API.                                                                    |
+| `teleport_audit_emit_events`                             | counter   | Teleport Audit Log  | Number of audit events emitted.                                                                    |
+| `teleport_audit_parquetlog_batch_processing_seconds`     | histogram | Teleport Audit Log  | Duration of processing single batch of events in parquet audit log.                                |
```

Comment:
Will readers of the docs know what "parquet audit log" means? "parquet" isn't mentioned in other places in the docs. 

Is there a plan to add instructions for configuring the Parquet audit log? Otherwise, I would make this something like, "Duration of processing a single batch of events in the Parquet-format audit log, if this is enabled".

This comment applies to the other new metric descriptions as well.

Reply:
yes, in separate PR I will add configuration of Parquet based audit log. I have updated description, but omitted "if this is enabled" because we don't use if for any other audit log or backend. PTAL.